Added notes on running nms on Linux.

bmurithi opened the pull request:
There are likely better ways to do this through declarative docker networking.

bmurithi committed:
Added notes on running nms on Linux.

xjtian reviewed (approved):
Looks legit, I'll import it

rpraveen commented on docs/readmes/basics/quick_start_guide.md:
```diff
@@ -111,3 +111,19 @@ HOST [magma/nms/fbcnms-projects/magmalte] $ ./scripts/dev_setup.sh
+HOST [magma]$ cd nms/fbcnms-projects/magmalte
+HOST [magma/nms/fbcnms-projects/magmalte] $ docker-compose up -d
+HOST [magma/nms/fbcnms-projects/magmalte] $ ./scripts/dev_setup.sh
+HOST[magma] docker network connect orc8r_default magmalte_magmalte_1
```
Nice! Can we add this command to ./scripts/dev_setup.sh so that its done automatically by the script? Also we can set the API_HOST to "orc8r_proxy_1:9443" in the Dockerfile so there is only one way of doing this for both OSX and Linux.

bmurithi replied:
@rpraveen made the changes - tested on a linux guest (Ubuntu 18.04).

Untested on Mac OS.

bmurithi committed:
Merge branch 'master' into linux-development-doc-notes

bmurithi committed:
Added docker network connect call to dev_setup.sh. Container naming.

- Added orcr8_proxy and magmalte_nms explicit container_names
  for predictable configuration.
